core/src/datayoga_core/blocks/map/tests/test_map.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from datayoga_core import result
+from datayoga_core.blocks.map.block import Block
+
+
+@pytest.mark.asyncio
+async def test_map_expression_jmspath():
+    block = Block(properties={"language": "jmespath",
```

Comment:
since the only argument the constructor takes is the `properties`, I think it's redundant to pass it by its name.

Reply:
pylint (or one of the other linters) wasn't happy without explicit mention. It is also more durable if we add params later.

Reply:
OK, I don't understand what's the problem with passing an argument by position here, but we can leave it as it more readable.